Assets/Scripts/Data Model/LayerManager.cs

```diff
@@ -1,948 +1,950 @@
 
-	//private static void GetNewLayerMeta(int layerID, AddNewLayerCallback addNewLayerCallback)
-	//{
-	//	NetworkForm form = new NetworkForm();
+		//	GetNewLayerMeta(layerID, addNewLayerCallback);
+		//}
 
-	//	ServerCommunication.DoRequest(Server.LayerMeta(layerID), form, (www2) => { handleGetNewLayerMetaCallback(www2, addNewLayerCallback); });
-	//}
+		//private static void GetNewLayerMeta(int layerID, AddNewLayerCallback addNewLayerCallback)
+		//{
+		//	NetworkForm form = new NetworkForm();
 
-	//private static void handleGetNewLayerMetaCallback(UnityWebRequest www, AddNewLayerCallback addNewLayerCallback)
-	//{
-	//	List<LayerMeta> layerMeta = LayerInfo.Load(www);
+		//	ServerCommunication.DoRequest(Server.LayerMeta(layerID), form, (www2) => { handleGetNewLayerMetaCallback(www2, addNewLayerCallback); });
+		//}
 
-	//	int layerID = layerMeta[0].layer_id;
+		//private static void handleGetNewLayerMetaCallback(UnityWebRequest www, AddNewLayerCallback addNewLayerCallback)
+		//{
+		//	List<LayerMeta> layerMeta = LayerInfo.Load(www);
 
-	//	LoadLayer(layerID, new List<SubEntityObject>());
+		//	int layerID = layerMeta[0].layer_id;
 
-	//	AbstractLayer layer = GetLayerByID(layerID);
-	//	addNewLayerCallback(layer);
-	//}
+		//	LoadLayer(layerID, new List<SubEntityObject>());
 
-	public static void ResetAll()
-	{
-		layers.Clear();
-		loadedLayers.Clear();
-		visibleLayers.Clear();
-		finishedImporting = false;
-	}
+		//	AbstractLayer layer = GetLayerByID(layerID);
+		//	addNewLayerCallback(layer);
+		//}
 
-	//public static void MergeAllVisibleLayers()
-	//{
-	//	List<AbstractLayer> mergeLayers = new List<AbstractLayer>(visibleLayers);
-	//	for (int i = mergeLayers.Count - 1; i >= 0; --i)
-	//	{
-	//		if (!mergeLayers[i].Selectable) { mergeLayers.RemoveAt(i); }
-	//	}
-
-	//	if (mergeLayers.Count < 2)
-	//	{
-	//		Debug.LogError("Unable to merge the visible layers: less than 2 layers selected");
-	//		return;
-	//	}
-
-	//	bool geoTypeDefined = false;
-	//	GeoType geoType = GeoType.polygon;
-	//	foreach (AbstractLayer layer in mergeLayers)
-	//	{
-	//		if (!geoTypeDefined) { geoType = layer.GetGeoType(); geoTypeDefined = true; }
-	//		else
-	//		{
-	//			if (geoType != layer.GetGeoType())
-	//			{
-	//				Debug.LogError("Unable to merge the visible layers: not all layers have the same type");
-	//				return;
-	//			}
-	//		}
-	//	}
-
-	//	Debug.Log("Merging visible layers");
-	//	AddNewLayer("Merged layer", geoType, (layer) => mergedLayerCreated(layer, mergeLayers));
-	//}
-
-	private static void AddToCategories(string category, string subcategory)
-	{
-		if (!categorySubcategories.ContainsKey(category))
+		public static void ResetAll()
 		{
-			categorySubcategories.Add(category, new List<string>());
+			layers.Clear();
+			loadedLayers.Clear();
+			visibleLayers.Clear();
+			finishedImporting = false;
 		}
 
-		if (!categorySubcategories[category].Contains(subcategory))
+		//public static void MergeAllVisibleLayers()
```

Comment:
Commented code, as always see if it makes sense to leave it there for the future, otherwise remove it! :)